### PR TITLE
Add support for archive zone with single site

### PR DESCRIPTION
**Pull request by mkasturi18**
Signed-off-by: MadhaviKasturi <mkasturi@redhat.com>

# Description
Add support for archive zone with single site 
Polarion # CEPH-83573386

Pass logs - http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-YWJO0L/ 

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

**Comment by anrao19** [comment]
@mkasturi18, If this suite will be part of pipeline, can we please update 5.x metadata file

**Comment by mkasturi18** [comment]
@anrao19 , thank you, added suite to pipeline

File: pipeline/metadata/5.3.yaml [hunk 1]
```diff
@@ -1152,3 +1152,19 @@
...
+  inventory:
+    openstack: "conf/inventory/rhel-8-latest.yaml"
+    ibmc: "conf/inventory/ibm-vpc-rhel-8-latest.yaml"
+  metadata:
+    - schedule
+    - tier-2
+    - openstack
+    - ibmc
+    - rgw
+    - stage-2
```
Comment (ckulal) [inline pipeline/metadata/5.3.yaml:1170]: for schedule run looks like there are already 7 suites under tier-2 stage-2,
Can we please choose some other stage?

File: suites/pacific/rgw/tier-2_rgw_archive.yaml [hunk 2]
```diff
@@ -0,0 +1,233 @@
+# This test is to verify the archive zone with single site
```
Comment (Ameenasuhani) [inline suites/pacific/rgw/tier-2_rgw_archive.yaml:1]: Please add a doc string indicating which conf is used and brief overview

File: pipeline/metadata/5.3.yaml (hunk 1 above)
Comment (mkasturi18) [reply pipeline/metadata/5.3.yaml]: @ckulal added to tier-2 stage-3

File: suites/pacific/rgw/tier-2_rgw_archive.yaml (hunk 2 above)
Comment (mkasturi18) [reply suites/pacific/rgw/tier-2_rgw_archive.yaml:1]: @Ameenasuhani the doc string and description added

**Review by ckulal — APPROVED** [review]

**Review by anrao19 — APPROVED** [review]

**Review by TejasC88 — APPROVED** [review]

**Pull request merged**